crates/topos/tests/snapshots/regtest__regtest_spam_help_display.snap

```diff
@@ -33,6 +33,12 @@ Options:
           Socket of the opentelemetry agent endpoint. If not provided open telemetry will not be used [env: TOPOS_OTLP_AGENT=]
       --otlp-service-name <OTLP_SERVICE_NAME>
           Otlp service name. If not provided open telemetry will not be used [env: TOPOS_OTLP_SERVICE_NAME=]
+      --benchmark
```

Comment:
In the PR description I read `--kubernetes` but it is actually `--benchmark` right? I mean this here is correct and the PR description is wrong?

Reply:
Is the spammer actually functional without this flag? Not sure what the use case for the cert spammer outside of the kubernetes stuff we're building is. If there is none, then I think it can be the default and save us some API surface to care about.

Reply:
Yeah that's a good point. I was under the assumption that we are still using `regtest spam` in the `local-erc20-infra` for our `e2e` tests. But it doesn't seem like the case.

Technically we can run `regtest spam` in a Docker environment with fixed `IP`s as well, and I wouldn't want to remove that feature. 

WDYT, @Freyskeyd?

Reply:
We need to keep the possibility to run it locally I guess